python/flyft/state.py

```diff
@@ -51,36 +51,50 @@ def __init__(self, L, shape, origin=0.):
         super().__init__(L, shape, origin)
 
     @property
-    def coordinates(self):
-        if not hasattr(self, '_coordinates'):
-            self._coordinates = np.array([self._self.coordinate(i) for i in range(self.shape)])
-        return self._coordinates
+    def center(self):
+        if not hasattr(self, '_center'):
+            self._center = np.array([self._self.center(i) for i in range(self.shape)])
+        return self._center
```

Comment:
We currently pluralize anything that's an array, so this should be:
```suggestion
    def centers(self):
        if not hasattr(self, '_centers'):
            self._centers = np.array([self._self.center(i) for i in range(self.shape)])
        return self._centers
```